backend/app/models/record/record.py

```diff
@@ -0,0 +1,31 @@
+from pydantic import BaseModel
+
+
+class AbstractInfo(BaseModel):
+    """Pydantic Model that holds the Abstract Information of a Wikipedia Article."""
+
+    title: str = ""
+    abstract: str = ""
+    url: str = ""
+
+
+class Categories(BaseModel):
+    """Pydantic Model to hold a category of a Wikipedia Article."""
+
+    text: str = ""
+    link: str = ""
+
+
+class ExternalLink(BaseModel):
+    """Pydantic Model to hold an external link of a Wikipedia Article."""
+
+    title: str = ""
+    link: str = ""
+
+
+class Record(BaseModel):
+    """Pydantic Model to hold a record."""
+
+    abstract_info: AbstractInfo | None = None
```

Comment:
Per previous comment you can just inline the `AbstractInfo` fields here instead of having the separate abstract structure. A record has-a title makes more sense than a record has-a abstract has-a title. The "abstract" stuff is an artifact of Wikipedia formats rather than something you want to model here to be shared with other input sources.

Reply:
When in doubt it's useful to look at schema.org precedents like https://schema.org/Article